asapdiscovery-data/asapdiscovery/data/scripts/write_cmpd_to_frag_dict.py

```diff
@@ -35,8 +30,6 @@ def get_args():
         "-o",
         "--out_yaml",
         default=os.path.join(
-            repo_path,
-            "data",
             "cmpd_to_frag.yaml",
         ),
```

Comment:
I'm going to change this to no longer be a default path and instead just put a suggested path in the example

Reply:
I agree, this is better. Thanks!